tcms/testplans/templates/testplans/get.html

```diff
@@ -140,6 +140,32 @@ <h2 class="card-pf-title">
     </div>
 </template>
 
+<template id="test_case_row_details">
+    <div class="list-view-pf-left">
+        <a href="/case/${testCase.id}" class="js-test-case-link"></a>
+    </div>
+    <div class="list-view-pf-body">
+        <div class="list-view-pf-additional-info">
+            <div class="list-view-pf-additional-info-item">
+                <span class="fa fa-hourglass"></span>
+                {% trans 'Priority' %}:&nbsp;<span class="js-test-case-priority"></span>
```

Comment:
We don't need the text label and space between the icon and the actual value. If there is ambiguity in some cases we can use a title attribute to make things more clear but for now there isn't. 

Icons make the interface cleaner and easier to look at.